### ISSUE TITLE
Output running haml-lint 

### ISSUE THREAD

**Issue by yosvel-quintero** [brigade/haml-lint#105]
### Versions:

* ruby 2.1.1
* overcommit 0.27.0
* haml-lint 0.13.0

### Console output:

```
git commit -m 'asdf'
Running pre-commit hooks
Analyzing with haml-lint...................................[HamlLint] FAILED
Hook raised unexpected error
Unexpected output: unable to determine line number or type of error/warning for message 'undefined method `type' for false:FalseClass'
/Users/userName/.rvm/rubies/ruby-2.1.1/lib/ruby/gems/2.1.0/gems/overcommit-0.27.0/lib/overcommit/hook/pre_commit/base.rb:30:in `block in extract_messages'
/Users/userName/.rvm/rubies/ruby-2.1.1/lib/ruby/gems/2.1.0/gems/overcommit-0.27.0/lib/overcommit/hook/pre_commit/base.rb:28:in `map'
/Users/userName/.rvm/rubies/ruby-2.1.1/lib/ruby/gems/2.1.0/gems/overcommit-0.27.0/lib/overcommit/hook/pre_commit/base.rb:28:in `extract_messages'
/Users/userName/.rvm/rubies/ruby-2.1.1/lib/ruby/gems/2.1.0/gems/overcommit-0.27.0/lib/overcommit/hook/pre_commit/haml_lint.rb:14:in `run'
/Users/userName/.rvm/rubies/ruby-2.1.1/lib/ruby/gems/2.1.0/gems/overcommit-0.27.0/lib/overcommit/hook/base.rb:45:in `block in run_and_transform'
/Users/userName/.rvm/rubies/ruby-2.1.1/lib/ruby/gems/2.1.0/gems/overcommit-0.27.0/lib/overcommit/utils.rb:211:in `with_environment'
/Users/userName/.rvm/rubies/ruby-2.1.1/lib/ruby/gems/2.1.0/gems/overcommit-0.27.0/lib/overcommit/hook/base.rb:45:in `run_and_transform'
/Users/userName/.rvm/rubies/ruby-2.1.1/lib/ruby/gems/2.1.0/gems/overcommit-0.27.0/lib/overcommit/hook_runner.rb:107:in `block in run_hook'
/Users/userName/.rvm/rubies/ruby-2.1.1/lib/ruby/gems/2.1.0/gems/overcommit-0.27.0/lib/overcommit/interrupt_handler.rb:63:in `disable_until_finished_or_interrupted'
/Users/userName/.rvm/rubies/ruby-2.1.1/lib/ruby/gems/2.1.0/gems/overcommit-0.27.0/lib/overcommit/hook_runner.rb:106:in `run_hook'
/Users/userName/.rvm/rubies/ruby-2.1.1/lib/ruby/gems/2.1.0/gems/overcommit-0.27.0/lib/overcommit/hook_runner.rb:59:in `block in run_hooks'
/Users/userName/.rvm/rubies/ruby-2.1.1/lib/ruby/gems/2.1.0/gems/overcommit-0.27.0/lib/overcommit/hook_runner.rb:58:in `each'
/Users/userName/.rvm/rubies/ruby-2.1.1/lib/ruby/gems/2.1.0/gems/overcommit-0.27.0/lib/overcommit/hook_runner.rb:58:in `run_hooks'
/Users/userName/.rvm/rubies/ruby-2.1.1/lib/ruby/gems/2.1.0/gems/overcommit-0.27.0/lib/overcommit/hook_runner.rb:39:in `block in run'
/Users/userName/.rvm/rubies/ruby-2.1.1/lib/ruby/gems/2.1.0/gems/overcommit-0.27.0/lib/overcommit/interrupt_handler.rb:84:in `isolate_from_interrupts'
/Users/userName/.rvm/rubies/ruby-2.1.1/lib/ruby/gems/2.1.0/gems/overcommit-0.27.0/lib/overcommit/hook_runner.rb:24:in `run'
.git/hooks/pre-commit:69:in `<main>'
Analyzing with RuboCop......................................[RuboCop] OK
Running JsTranslations...............................[JsTranslations] OK
✗ One or more pre-commit hooks failed
```

**Comment by sds** [comment]
Hey @yosvel-quintero,

Please update to the latest version of `haml-lint` and `overcommit`, and let us know if you still see this issue.
